utils.sh

```diff
@@ -57,7 +57,6 @@ function configure_chronyd() {
   else
     sudo firewall-cmd --permanent --add-service=ntp
   fi
-  sudo firewall-cmd --reload
```

Comment:
If this gets removed the rule won't take effect.  Line 58 will need to be run again without `--permanent`:
```
       --permanent
           The permanent option --permanent can be used to set options permanently. These changes are not effective immediately, only after service restart/reload or system reboot. Without the --permanent
           option, a change will only be part of the runtime configuration.
```


```
sudo firewall-cmd --permanent --add-service=ntp
sudo firewall-cmd --add-service=ntp
```

Reply:
Oh I didn't realize `--permanent` disabled the runtime configuration, I guess we'll have to do two calls